web3/method.py

```diff
@@ -104,21 +113,7 @@ def method_selector_fn(self):
             return lambda *_: self.json_rpc_method
         raise ValueError("``json_rpc_method`` config invalid.  May be a string or function")
 
-    def get_formatters(self, method_string):
-        """Lookup the request formatters for the rpc_method
-
-        The lookup_fn output is expected to be a 2 length tuple of lists of
-        the request and output formatters, respectively.
-        """
-        formatters = self.formatter_lookup_fn(method_string)
-        return formatters or get_default_formatters()
-
-    def input_munger(self, val):
-        try:
-            module, args, kwargs = val
-        except TypeError:
-            raise ValueError("input_munger expects a 3-tuple")
-
+    def input_munger(self, module, args, kwargs):
         # TODO: Create friendly error output.
         mungers_iter = iter(self.mungers)
         root_munger = next(mungers_iter)
```

Comment:
I know this code wasn't changed, but it strikes me as something that is in need of a nice comment explaining what it is doing as it is quite cryptic at-a-glance.